resources/web/local/qwebchannel.js

```diff
@@ -0,0 +1,448 @@
+/****************************************************************************

```

Comment:
Shouldn't we remove this file? It seems useless now, isn't it?

Reply:
> Nothing catches the eye, only a remaining reference. The branch compiles fine on ubuntu 20 and the robot windows open up correctly.
> 
> Running the test suite locally this test also failed:
> 
> `FAILURE with speaker: wb_speaker_is_speaking should return true when TTS is running.`
> 
> tried twice and happened both times, I don't know if it's related.

It is related to openAL, #4047

Reply:
It is but the file is still include in the qt-5-12 folder package on the server. So each time we make Webots this file will be add to the `resources/web/local `. Because we will move to qt6, I suggest that we keep the file the time to move to qt6 because whatever it will be add when downloading the qt5 dependency.